cmd/wallet-web/src/pages/mixins/handler/waci.js

```diff
@@ -19,8 +19,8 @@ const base64url = require('base64url');
  */
 export class WACIRedirectHandler {
   constructor(oob, redirect) {
-    this.oob = JSON.parse(base64url.decode(oob));
-    this.redirect = redirect ? base64url.decode(redirect) : redirect;
+    this.oob = JSON.parse(Base64.decode(oob));
```

Comment:
we need base64url decode here

Reply:
link to the spec - https://identity.foundation/didcomm-messaging/spec/#standard-message-encoding 

> encodedplaintextjwm = b64urlencode(<plaintextjwm>)

Reply:
looks like library understands url vs non-url encoding in same function

from the library doc - https://github.com/dankogai/js-base64#synopsis

> Base64.decode(      '5bCP6aO85by+');// 小飼弾
> // note .decodeURI() is unnecessary since it accepts both flavors
> Base64.decode(      '5bCP6aO85by-');// 小飼弾

Reply:
Created an issue - https://github.com/trustbloc/wallet/issues/1235